packages/ken-js/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
why do we need this file, instead of using bs-ken?

Reply:
Yes, sure, give me a pair of miniutes